torch/_export/serde/schema.py

```diff
@@ -154,15 +148,23 @@ class GraphSignature:
     user_inputs: List[str]
     user_outputs: List[str]
     buffers_to_mutate: Dict[str, str]
+    backward_signature: Optional[BackwardSignature]
+
+
+@dataclass
+class CallSpec:
     in_spec: str
     out_spec: str
-    backward_signature: Optional[BackwardSignature]
 
 
 @dataclass
 class GraphModule:
     graph: Graph
-    buffers: Dict[str, TensorMeta]
-    parameters: Dict[str, TensorMeta]
-    metadata: Dict[str, str]
+    opset_version: Dict[str, int]
```

Comment:
@angelayi I think it makes more sense to me if we put opset_version in ExportedProgram, and put signature in GraphModule. The rationale is to closely mirror the schema in sigmoid.